modules/seo-tools/jetpack-seo.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * An SEO expert walks into a bar, bars, pub, public house, Irish pub, drinks, beer, wine, liquor, Grey Goose, Cristal...
+ */
+class Jetpack_SEO {
+	public function __construct() {
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	public function init() {
+		if ( apply_filters( 'jetpack_seo_meta_tags', true ) ) {
+			add_action( 'wp_head', array( $this, 'meta_tags' ) );
```

Comment:
In my opinion, the more fine-grained control we provide, the better. But in this case as @vindl pointed out it may not be the best decision to bundle all tags into one filter. Perhaps we need a second filter to change the way we output non-OG tags? And we can add additional on/off filters to short-circuit them without any processing.